go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/aws/aws-sdk-go v1.49.19
 	github.com/aws/aws-sdk-go-v2 v1.24.1
 	github.com/aws/aws-sdk-go-v2/config v1.26.6
-	github.com/aws/aws-sdk-go-v2/credentials v1.16.16
+	github.com/aws/aws-sdk-go-v2/credentials v1.16.16 // indirect
```

Comment:
Curious what is this comment for?

Reply:
It changed to indirect because I removed evergreen's usage of static credentials, but other packages like `pail` that evergreen imports do still use it.

Reply:
Gotcha, thanks